.circleci/config.yml

```diff
@@ -178,6 +178,18 @@ workflows:
       - build-deb-package:
           name: debian11
           image: debian:bullseye
+      - build-rpm-package:
+          name: rockylinux8.4
+          image: rockylinux/rockylinux:8.4
+      - build-rpm-package:
+          name: almalinux8.4
+          image: almalinux:8.4
```

Comment:
I think we can drop these in next revision, for the minor version

Reply:
Done